app/assets/javascripts/active_admin/application.js.coffee

```diff
@@ -1,10 +1,19 @@
 # Initializers
 $(document).on 'ready page:load', ->
   # jQuery datepickers (also evaluates dynamically added HTML)
-  $(document).on 'focus', '.datepicker:not(.hasDatepicker)', ->
+  datepicker_selector = 'input.datepicker:not(.hasDatepicker)'
+
+  $(document).on 'focus', datepicker_selector, ->
```

Comment:
Please put this (Line 4-6) into one line, I know @houndci say something different